apps/aecore/test/aecore_txs_SUITE.erl

```diff
@@ -183,6 +185,56 @@ missing_tx_gossip(Config) ->
 
     ok.
 
+check_coinbase_validation(Config) ->
+    aecore_suite_utils:start_node(dev1, Config),
+    aecore_suite_utils:start_node(dev2, Config),
+
+    N1 = aecore_suite_utils:node_name(dev1),
+    N2 = aecore_suite_utils:node_name(dev2),
+
+    aecore_suite_utils:connect(N1),
+    aecore_suite_utils:connect(N2),
+
+    %% Both nodes are up, now turn off gossiping of TXs
+    %% Also (virtually) disable ping
+    rpc:call(N1, aec_sync, gossip_txs, [false], 5000),
+    rpc:call(N2, aec_sync, gossip_txs, [false], 5000),
+    rpc:call(N1, application, set_env, [aecore, ping_interval, 1000000], 5000),
+    rpc:call(N2, application, set_env, [aecore, ping_interval, 1000000], 5000),
+
+    %% Ping interval was 500 ms, wait that long
+    timer:sleep(2 * 500),
```

Comment:
This test uses dev1 and dev2 nodes. dev2 has ping interval 500ms buried in test config file (would be ideally dynamically fetched by test, though debatable because under control of test), dev1 has none so shall pick default 2 minutes?

I need to understand better expectation of this test re dev1.

Reply:
Meant to be addressed by PR #1527 .